.clang-format

```diff
@@ -6,13 +6,14 @@ AlignConsecutiveAssignments: true
 AlignConsecutiveDeclarations: true
 # AlignEscapedNewlinesLeft: false
 AlignOperands:   true
+AllowShortBlocksOnASingleLine: Empty
 AlignTrailingComments: true
 # AllowAllArgumentsOnNextLine: true    # Requires clang-format v9 and higher
 # AllowAllConstructorInitializersOnNextLine: false
 AllowAllParametersOfDeclarationOnNextLine: true
 AllowShortBlocksOnASingleLine: false
 # AllowShortCaseLabelsOnASingleLine: false
-# AllowShortFunctionsOnASingleLine: All
+AllowShortFunctionsOnASingleLine: Empty
```

Comment:
I'd prefer `Inline` for this option. This keeps short functions in one line, when defined inside a class.

Reply:
Changed to Inline